routes/web.php

```diff
@@ -13,6 +13,10 @@
 Route::get('/companies/{slug}', [CompanyController::class, 'show'])
     ->name('companies.show');
 
+Route::post('/companies/{slug}/reviews', [CompanyController::class, 'storeReview'])
```

Comment:
Use {company} instead of slug. Slug as a string should only be used when we need to fetch more info about the company.